app/controllers/assignments_controller.rb

```diff
@@ -581,6 +581,23 @@ def switch_assignment
     end
   end
 
+  def set_boolean_graders_options
+    unless %w[assign_graders_to_criteria
```

Comment:
(This is the part I was referring to above where you should be able to use `params.permit`.